crates/forge/tests/integration/gas.rs

```diff
@@ -532,9 +532,9 @@ fn multiple_storage_writes_cost() {
     let result = run_test_case(&test);
 
     assert_passed!(result);
-    // 3 = gas cost of steps
+    // 4 = gas cost of steps
```

Comment:
Are you sure? Nothing else changed and steps were calculated correctly, prob another builtin (that was previously calculated incorrectly) is responsible for this (see [this](https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/fee-mechanism/#gas_cost_per_cairo_step_or_builtin_step) and the formula above)

Reply:
The cost has increased, because now the syscalls is counted twice, instead of once [here](https://github.com/starkware-libs/blockifier/blob/8f423a2f5c351e4c22c7350b08afd8082fb0fa9f/crates/blockifier/src/fee/os_usage.rs#L46).

Reply:
True, that means that #1631 is higher priority than I thought